mlrun/api/utils/clients/log_collector.py

```diff
@@ -214,14 +215,52 @@ async def stop_logs(
         :return: None
         """
 
-        request = self._log_collector_pb2.StopLogRequest(
+        request = self._log_collector_pb2.StopLogsRequest(
             project=project, runUIDs=run_uids
         )
 
-        response = await self._call("StopLog", request)
+        response = await self._call("StopLogs", request)
         if not response.success:
             msg = "Failed to stop logs"
             if raise_on_error:
+                if response.errorCode == LogCollectorErrorCode.ErrCodeBadRequest:
+                    raise mlrun.errors.MLRunBadRequestError(
+                        f"{msg},error= {response.errorMessage}"
+                    )
+                raise mlrun.errors.MLRunInternalServerError(
+                    f"{msg},error= {response.errorMessage}"
+                )
```

Comment:
it seems like have a bit of a code dup so to have a more clean code, I would do a mapping between the err codes and have a method which raises the error depends on the err code from the log collector.